demisto_sdk/commands/validate/validators/IF_validators/IF109_invalid_required_field.py

```diff
@@ -0,0 +1,114 @@
+from __future__ import annotations
+
+from typing import Iterable, List, Union, cast
+
+from demisto_sdk.commands.common.constants import GitStatuses
+from demisto_sdk.commands.content_graph.objects.incident_field import IncidentField
+from demisto_sdk.commands.content_graph.objects.incident_type import IncidentType
+from demisto_sdk.commands.content_graph.objects.indicator_field import IndicatorField
+from demisto_sdk.commands.content_graph.objects.indicator_type import IndicatorType
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Union[IncidentField, IndicatorField, IncidentType, IndicatorType]
+
+
+class IsValidRequiredFieldValidator(BaseValidator[ContentTypes]):
+    error_code = "IF109"
+    description = "Checks if the incident field required field is valid."
+    rationale = "In case an incident field is required, newly added associated incident types should be new."
+    related_field = "required"
+    expected_git_statuses = [GitStatuses.MODIFIED, GitStatuses.ADDED]
```

Comment:
why did you remove the error message completely?
I'd rather have it with just {0} and format the returned message than remove it.

Reply:
cool, i'll return it